CWP/papers/roadmap/authors/address.txt

```diff
@@ -0,0 +1,112 @@
+ANL: High Energy Physics Division, Argonne National Laboratory, Argonne IL, USA
+BNL: Physics Department, Brookhaven National Laboratory, Upton NY, USA
+Caltech: California Institute of Technology, Pasadena, California, USA
+CBPF: Centro Brasileiro de Pesquisas Fisicas, Rio de Janeiro, Brazil
+CCIN2P3: Centre de Calcul de l’IN2P3, Villeurbanne, Lyon, France
+CERN: CERN, Geneva, Switzerland
+CIEMAT: Centro de Investigaciones Energéticas Medioambientales y Tecnológicas (CIEMAT), Madrid, Spain
+CHICAGO: Enrico Fermi Institute, University of Chicago, Chicago IL, USA
+CINVESTAV: Cinvestav, Mexico City, Mexico
+CNAF: Centro Nazionale Analisi Fotogrammi (CNAF), INFN, Bologna, Italy
+Cornell: Cornell University, Ithaca, USA
+CPHC: Center for High Performance Computing, Cape Town, South Africa
+DESY: Deutsches Elektronen-Synchrotron, Hamburg, Germany
+EPFL: Institute of Physics, Ecole Polytechnique Federale de Lausanne (EPFL), Lausanne, Switzerland
+ETHZurich: ETH Zurich - Institute for Particle Physics and Astrophysics (IPA), Zurich, Switzerland
+Fermilab: Fermi National Accelerator Laboratory, Batavia, USA
+HAMBURG: University of Hamburg, Hamburg, Germany
+HIP: Helsinki Institute of Physics, Helsinki, Finland
+IFAE: Institut de Física d’Altes Energies and Departament de Física de la Universitat Autònoma de Barcelona and ICREA, Barcelona, Spain
+IFCA: Instituto de Física de Cantabria (IFCA), CSIC-Universidad de Cantabria, Santander, Spain
+IFICValencia: Instituto de Fisica Corpuscular, Centro Mixto Universidad de Valencia - CSIC, Valencia, Spain
+IHEPCHINA: Institute of High Energy Physics, Chinese Academy of Sciences, Beijing
+INDIANA: Department of Physics, Indiana University, Bloomington IN, USA
+INFNBOLOGNA: INFN Sezione di Bologna, Università di Bologna, Bologna, Italy
+INFNFerrara: Universita e INFN, Ferrara, Ferrara, Italy
+INFNBari: INFN Sezione di Bari, Università di Bari, Politecnico di Bari, Bari, Italy
+INFNMilano: INFN Sezione di Milano-Bicocca, Milano, Italy
+INFNNapoli: INFN Sezione di Napoli, Università di Napoli, Napoli, Italy
+INFNPadova: INFN Sezione di Padova, Università di Padova b, Padova, Italy
+INFNPISA: INFN Sezione di Pisa, Università di Pisa, Scuola Normale Superiore di Pisa, Pisa, Italy
+INFNRoma: INFN Sezione di Roma I, Università La Sapienza, Roma, Italy
+INFNTRIESTE: INFN Sezione di Trieste, Università di Trieste, Trieste, Italy
+IPHC: Université de Strasbourg, CNRS, IPHC UMR 7178, F-67000 Strasbourg, France
+Irfu: DSM/IRFU (Institut de Recherches sur les Lois Fondamentales de l’Univers), CEA Saclay (Commissariat a l’Energie Atomique), Gif-sur-Yvette, France
```

Comment:
Commissariat à l'énergie atomique

Don't know if you consider it ok to drop the accent if capitalised

Reply:
Personally I'm old style and dropping it. I think that now people tends to keep it... Equal for me, I think we must as much as possible stick with what the institution wants. As said in an email, all these affiliations are copy/paste from experiment articles and I'm reluctant to change it... I'm afraid that Shangai is accent-sensitive and this is one of the reason official affiliation chose one or the other form...